code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -50,9 +51,9 @@
 		if(new_snailperson.dropItemToGround(bag)) //returns TRUE even if its null
 			new_snailperson.equip_to_slot_or_del(new /obj/item/storage/backpack/snail(new_snailperson), ITEM_SLOT_BACK)
 	new_snailperson.AddElement(/datum/element/snailcrawl)
-	new_snailperson.update_icons() //SKYRAT EDIT: Roundstart Snails
+	new_snailperson.update_icons() //SKYRAT EDIT: Roundstart Snails - This is to help with shell reskins.
 	//if(ishuman(new_snailperson)) //SKYRAT EDIT: Snails don't have exotic blood here!
-	//	update_mail_goodies(new_snailperson) //SKYRAT EDIT END
+	//	update_mail_goodies(new_snailperson) //SKYRAT EDIT END - Roundstart Snails - They don't need special mail goodies because of the above.
```

Comment:
Use `/*` and `*/` on the lines before and after to comment stuff out, like we do elsewhere in the code.